src/components/pages/accounts/AccountsTable.tsx

```diff
@@ -66,7 +66,7 @@ function getTreeTotals(
   const leaves: AccountsTableRow[] = [];
   current.childrenIds.forEach(childId => {
     const childAccount = accounts[childId];
-    if (!childAccount.hidden) {
+    if (!childAccount.hidden && childAccount.parentId === current.guid) {
```

Comment:
this is kind of a hack. The childrenIds is inconsistent for when we update the parent of an account as we don't remove it from the list of childrenIds of the old parent. If we want to set the correct behavior, we need to fetch the original account and do some extra operations. Going pragmatic for now